locales/ja.json

```diff
@@ -6,45 +6,45 @@
         "clear": "クリア"
     },
     "logout": {
-        "message": "Logging you out...",
-        "successMessage": "You've been logged out. See ya!"
+        "message": "ログアウト中...",
+        "successMessage": "ログアウトしました。"
     },
     "index": {
         "title": "COVID-19日本のワクチン接種キャンセル待ちリスト",
-        "massSite": "バウチャーをお持ちの18歳以上の方は、東京または大阪の集団予防接種サイトで予防接種を受けることができます。",
+        "massSite": "接種券をお持ちの18歳以上の方は、東京または大阪の大規模接種センターで予防接種を受けることができます。",
         "register": "登録サイトに移動します"
     },
     "add-clinic": {
-        "fillForm": "このフォームに記入して、キャンセル待機リストのある新しいクリニックをデータベースに追加します。",
+        "fillForm": "このフォームを記入し、キャンセル待ちリストのあるクリニックをデータベースに登録できます。",
         "romajiOnly": "ローマ字",
         "prefecture": "都道府県",
-        "city": "都市",
```

Comment:
This is for an input form to add the city. Thought we wanted 都市

Reply:
In broad terms Japan is divided administratively into prefectures (都道府県), then municipalities (市区町村). As findadoc.jp is now covering clinics nationally, and because there is a separate ward (区) category, I plumped for 市町村 here. I felt this would be most natural for native readers and in line with conventions. 都市 is not really used for addressing like this.

Reply:
I agree with @alexmalins here - if you look at similar Japanese websites they usually list this portion as 市区町村, as not everything qualifies as a 都市 technically. Since we cover 区 elsewhere, I think "市町村" makes sense.

Reply:
It would be most natural to name the three categories as something like prefecture (都道府県), municipality (市区町村) & neighborhood (町域 or 町・大字), I feel. It would take a small amount of work to bring the database entries in line with this, but happy to do this if it helps.

Reply:
Thanks for explaining. Was thinking 都市 because I used it for a form on another project, but it's clear you've considered the context and I trust your judgment.